test/common.sh

```diff
@@ -0,0 +1,276 @@
+
+#!/bin/bash
+
+CATALOG_TEST_SKIP_CLEANUP=true
+
+# Define a custom kubectl path if you like
+KUBECTL_CMD=${KUBECTL_CMD:-kubectl}
+
+TEST_RESULT_FILE=/tmp/test-result
+
+# Checks whether the given function exists.
+function function_exists() {
+  [[ "$(type -t $1)" == "function" ]]
+}
+
+# Set the return code that the test script will return.
+# Parameters: $1 - return code (0-255)
+function set_test_return_code() {
+  # kubetest teardown might fail and thus incorrectly report failure of the
+  # script, even if the tests pass.
+  # We store the real test result to return it later, ignoring any teardown
+  # failure in kubetest.
+  # TODO(adrcunha): Get rid of this workaround.
+  echo -n "$1"> ${TEST_RESULT_FILE}
+}
+
+function detect_changed_e2e_test() {
+    # check if any file from test/ directory is changed
+    echo ${CHANGED_FILES} |grep "test/" || true
+}
+
+function detect_new_changed_resources() {
```

Comment:
function name is unclear

Reply:
changed it to `print_changed_resources_to_console` now